### PR TITLE
Add ifermi

### PR DESCRIPTION
**Pull request by jan-janssen**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by jan-janssen** [comment]
@conda-forge/staged-recipes, ready for review

File: recipes/ifermi/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ifermi-{{ version }}.tar.gz
+  sha256: e350e4dafd95d5bb019af683d7b016cbb2535a62a43b46050d004e924d3664e8
+
+build:
+  number: 0
+  entry_points:
+    - ifermi = ifermi.cli:cli
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: True  # [win]
```
Comment (dopplershift) [inline recipes/ifermi/meta.yaml:18]: Upstream says "OS independent". I can't see the build log, why did this fail on Windows?

Comment (jan-janssen) [reply recipes/ifermi/meta.yaml]: `boltztrap2` is only available on Linux and Mac Os X - still you are right no arch is most likely fine.

**Comment by jan-janssen** [comment]
@dopplershift Can you take another look at this pull request?

**Review by dopplershift — APPROVED** [review]

**Comment by dopplershift** [comment]
Ignoring windows failure (due to missing dep)

**Pull request merged**